File: F1Levier/app/src/main/java/com/djira/f1levier/ui/participant/ParticipantViewModel.java
package com.djira.f1levier.ui.participant;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class ParticipantViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public ParticipantViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is participant fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: TP4/CourseCODEP25/app/src/main/java/com/utbm/nouassi/manou/coursecodep25/ui/participant/ParticipantViewModel.kt
package com.utbm.nouassi.manou.coursecodep25.ui.participant

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class ParticipantViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is gallery Fragment"
    }
    val text: LiveData<String> = _text
}